hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieDemo.java

```diff
@@ -110,9 +110,9 @@ private void setupDemo() throws Exception {
     executeCommandStringsInDocker(PRESTO_COORDINATOR, cmds);
 
     // copy presto sql files to presto coordinator
-    executePrestoCopyCommand( System.getProperty("user.dir") + "/.." + PRESTO_INPUT_TABLE_CHECK_RELATIVE_PATH, HDFS_DATA_DIR);
-    executePrestoCopyCommand( System.getProperty("user.dir") + "/.." + PRESTO_INPUT_BATCH1_RELATIVE_PATH, HDFS_DATA_DIR);
-    executePrestoCopyCommand( System.getProperty("user.dir") + "/.." + PRESTO_INPUT_BATCH2_RELATIVE_PATH, HDFS_DATA_DIR);
+    executePrestoCopyCommand(System.getProperty("user.dir") + "/.." + PRESTO_INPUT_TABLE_CHECK_RELATIVE_PATH, HDFS_DATA_DIR);
```

Comment:
may I know why we see just check style changes in few files? was there any change in check style recently ?

Reply:
Not really, it is still the checkstyle from incubator-hudi/style/checkstyle.xml. When I finish adding the doc and comments,I am not sure whether the changes I made can meet up the checkstyle requirements, so I scanned the entire project using the checkstyle plugin and found these checkstyle issues, so I 'corrected' them . Are theses changes in checkstyle right? should i revert them?